apps/user_ldap/lib/Access.php

```diff
@@ -528,7 +528,7 @@ public function dn2username($fdn, $ldapName = null) {
 	 */
 	public function dn2ocname($fdn, $ldapName = null, $isUser = true, &$newlyMapped = null, array $record = null) {
 		static $intermediates = [];
-		if (isset($intermediates[$isUser ? 'user-' : 'group-' . $fdn])) {
+		if (isset($intermediates[$isUser ? 'user-' . $fdn : 'group-' . $fdn])) {
```

Comment:
```suggestion
		if (isset($intermediates[($isUser ? 'user-' : 'group-') . $fdn])) {
```
Or make it more explicit?

Reply:
Done :)